examples/ES/learner.py

```diff
@@ -206,9 +203,6 @@ def _update_filter(self):
 
     def log_metrics(self, metrics):
         logger.info(metrics)
-        self.csv_logger.log_dict(metrics)
         for k, v in metrics.items():
-            tensorboard.add_scalar(k, v, self.sample_total_steps)
-
-    def close(self):
-        self.csv_logger.close()
+            if value is not None:
```

Comment:
where is the definition of `value`? It should be `v`

Reply:
fixed